share/locale/en.json

```diff
@@ -1,165 +1,165 @@
+    "settings_saved": "Settings saved in {}",
     "settings_error_unknown": "Can't connect to Tor controller because the settings don't make sense.",
-    "settings_error_automatic": "Can't connect to Tor controller. Is Tor Browser running in the background? If you don't have it you can get it from:\nhttps://www.torproject.org/.",
+    "settings_error_automatic": "Could not connect to the Tor controller. The Tor Browser (available from https://www.torproject.org/)\nneeds to be running in the background.",
     "settings_error_socket_port": "Can't connect to Tor controller on {}:{}.",
-    "settings_error_socket_file": "Can't connect to Tor controller using socket file {}.",
+    "settings_error_socket_file": "Can't connect to Tor controller using the socket file {}.",
     "settings_error_auth": "Connected to {}:{}, but can't authenticate. Maybe this isn't a Tor controller?",
-    "settings_error_missing_password": "Connected to Tor controller, but it requires a password to authenticate.",
-    "settings_error_unreadable_cookie_file": "Connected to Tor controller, but can't authenticate because your password may be wrong, and your user lacks permission to read the cookie file.",
-    "settings_error_bundled_tor_not_supported": "Use of the Tor version bundled with OnionShare is not supported when using developer mode on Windows or macOS.",
-    "settings_error_bundled_tor_timeout": "Connecting to Tor is taking too long. Maybe your computer is offline, or your system clock isn't accurate.",
-    "settings_error_bundled_tor_canceled": "The Tor process closed before it could finish connecting.",
+    "settings_error_missing_password": "Connected to Tor controller, type in the password to authenticate.",
+    "settings_error_unreadable_cookie_file": "Connected to the Tor controller, but can't authenticate. Maybe your password is wrong, or your user lacks permission to read the cookie file?",
+    "settings_error_bundled_tor_not_supported": "Useing the Tor version that comes with OnionShare does not work in developer mode on Windows or macOS.",
+    "settings_error_bundled_tor_timeout": "Connecting to Tor is taking too long. Maybe your computer is offline, or your system clock isn't accurate?",
+    "settings_error_bundled_tor_canceled": "The Tor process closed before a connection was established.",
     "settings_error_bundled_tor_broken": "OnionShare could not connect to Tor in the background:\n{}",
-    "settings_test_success": "Congratulations, OnionShare can connect to the Tor controller.\n\nTor version: {}\nSupports ephemeral onion services: {}\nSupports stealth onion services: {}",
-    "error_tor_protocol_error": "Could not communicate with the Tor controller.\nIf you're using Whonix, check out https://www.whonix.org/wiki/onionshare to make OnionShare work.",
+    "settings_test_success": "Connected to the Tor controller.\n\nTor version: {}\nSupports ephemeral onion services: {}.\nSupports onion services that are stealthy: {}.",
```

Comment:
All good, except the last part should just be: "Supports client authentication: {}" instead of "Supports onion services that are stealthy: {}"